tasks/main.yml

```diff
@@ -113,6 +110,13 @@
   notify: verify config and restart httpd
   when: _use_tls
 
+- name: remove obsolete files
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - "{{ _vhost_confdir }}/python.conf"
+
```

Comment:
Missing a notify to reload the server.

I would also like to see a comment to ay "remove this after $somedate" so we do not carry that for ever.